test/WormholeReceiver.t.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {Test} from "forge-std/Test.sol";
+
+import {WormholeReceiver} from "src/WormholeReceiver.sol";
+import {Constants} from "test/Constants.sol";
+
+contract WormholeReceiverTestHarness is WormholeReceiver {
+  constructor(address _relayer) WormholeReceiver(_relayer) {}
+  function receiveWormholeMessages(
+    bytes memory payload,
+    bytes[] memory additionalVaas,
+    bytes32 sourceAddress,
+    uint16 sourceChain,
+    bytes32 deliveryHash
+  ) public virtual override {}
+
+  function modifierTest() public onlyRelayer {}
+}
+
+contract OnlyRelayerTest is Test, Constants {
```

Comment:
(arbitrary comment)

Constructor sets private args so I couldn't test

Reply:
This begs the question, why is the Wormhole relayer private? Seems it would be fine for it to be public + immutable, or am I missing something?

Reply:
The main reason I made it private was because it conflicts with another instantiation of WORMHOLE_RELAYER in the inheritance chain. I guess this could be solved by abstracting that variable into another class?

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/26/commits/a430851e348915b3dbd7fe57720583dfaa170289